azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageTypeDaoImpl.java

```diff
@@ -171,4 +235,47 @@ public int createImageType(ImageType imageType) {
       return imageTypes;
     }
   }
+
+  /**
+   * ResultSetHandler implementation class for fetching image type
+   */
+  public static class FetchImageOwnershipHandler implements ResultSetHandler<List<ImageOwnership>> {
+
+    private static final String FETCH_IMAGE_OWNERSHIP_BY_IMAGE_TYPE_NAME =
+        "SELECT it.name, io.id, io.owner, io.role, io.created_by, io.created_on, io.modified_by, "
+            + "io.modified_on FROM image_types it, image_ownerships io  WHERE it.id = io.type_id "
+            + "and it.name = ?";
+    private static final String FETCH_IMAGE_OWNERSHIP_BY_ID =
```

Comment:
This query is not used anywhere. Can you please remove it?

Reply:
Done.